assets/src/edit-story/components/checklist/checks/pageTooMuchText.js

```diff
@@ -48,26 +62,50 @@ export function PageTooMuchText() {
     () => filterStoryPages(story, pageTooMuchText),
     [story]
   );
-  return failingPages.length > 0
-    ? // <ChecklistCard
-      //   title={__('Reduce amount of text on pages', 'web-stories')}
-      //   titleProps={{
-      //     onClick: () => {
-      //       /* perform highlight here */
-      //     },
-      //   }}
-      //   footer={
-      //     <Text size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}>
-      //       {__('Keep text to max 200 characters per page.', 'web-stories')}
-      //       <Link
-      //         href={'#' /* figure out what this links to */}
-      //         size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}
-      //       >
-      //         {'Learn more'}
-      //       </Link>
-      //     </Text>
-      //   }
-      // />
-      null
-    : null;
+  const setHighlights = useHighlights(({ setHighlights }) => setHighlights);
+  return failingPages.length > 0 ? (
+    <ChecklistCard
+      title={DESIGN_COPY.tooMuchPageText.title}
+      cardType={
+        failingPages.length > 1
+          ? CARD_TYPE.MULTIPLE_ISSUE
+          : CARD_TYPE.SINGLE_ISSUE
+      }
+      thumbnailCount={failingPages.length}
+      thumbnail={
+        <>
+          {failingPages.map((page) => (
+            <Thumbnail
+              key={page.id}
+              onClick={() => {
+                setHighlights({
+                  pageId: page.id,
+                });
+              }}
+              type={THUMBNAIL_TYPES.PAGE}
+              displayBackground={
+                <PagePreview
+                  page={page}
+                  width={THUMBNAIL_DIMENSIONS.WIDTH}
+                  height={THUMBNAIL_DIMENSIONS.HEIGHT}
+                  as="div"
+                />
+              }
+              aria-label="my helper text describing this thumbnail image"
```

Comment:
```suggestion
              aria-label={__('thumbnail of page related to checklist issue", "web-stories")} 
```

Or something like that?

Reply:
ope. forgot to update this. thanks for pointing out